src/main/kotlin/com/terraformation/backend/accelerator/api/ApplicationsController.kt

```diff
@@ -88,6 +93,19 @@ class ApplicationsController(
         deliverables.map { ApplicationDeliverablePayload(it) })
   }
 
+  @GetMapping("/{applicationId}/geoJson", produces = ["application/geo+json"])
+  @Operation(summary = "Get GeoJSON for an application")
+  fun getApplicationGeoJson(
+      @PathVariable applicationId: ApplicationId
+  ): ResponseEntity<InputStreamResource> {
```

Comment:
If at some point we want to support exporting in other formats, we can maybe consider making this a less GeoJSON-specific API the way, e.g., the search API looks at the `Accept` HTTP header to tell if it should return a CSV file.

Not saying we should do that now, but maybe we should make the URL suffix something like `boundary` or `export` instead of `geoJson`.

Reply:
Agreed! I did capture this in the JIRA ticket I think. But let me make sure to have it in the backlog.